src/Storages/MergeTree/KeyCondition.h

```diff
@@ -222,10 +220,6 @@ class KeyCondition
         };
         std::optional<MultiColumnsFunctionDescription> point_in_polygon_column_description;
 
-        using Point = boost::geometry::model::d2::point_xy<Float64>;
-        using Polygon = boost::geometry::model::polygon<Point>;
-        Polygon polygon;
```

Comment:
We should keep the polygon which is used when checking.

Reply:
Thanks, that was blind changes. Please continue...